### PR TITLE
Minor fixes to docker installation docu

### PR DESCRIPTION
**Pull request by DanielMorLoz**
Hi team,

I've added a new step to start docker before it is used because it comes inactive when you install it.
Also, I've manually set each step's number because it was shown "1" everywhere.

Regards.

### CODE REVIEW

File: source/docker/wazuh-container.rst [hunk 1]
```diff
@@ -99,7 +99,17 @@ Usage
 
       $ git clone https://github.com/wazuh/wazuh-docker.git -b 3.9.0_6.7.1 --single-branch
 
-#. Start Wazuh, Elastic Stack and Nginx using `docker-compose`. From the directory where you have the ``docker-compose.yml`` file:
+2. Start the docker service:
```
Comment (manuasir) [inline source/docker/wazuh-container.rst]: Please, change to `Docker` with uppercase.

Comment (DanielMorLoz) [reply source/docker/wazuh-container.rst]: Sorry for the delay, it is changed already.

**Comment by snaow** [comment]
Probably the best place to start the Docker service is, in the Docker installation guide:

https://documentation.wazuh.com/current/docker/docker-installation.html

Let's change it, please.

**Comment by DanielMorLoz** [comment]
Sorry for the delay,

The change proposed by @snaow has been done already.

**Comment by MiguelCasaresRobles** [comment]
Invalid

**Pull request closed without merging**